lib/model/shape.js

```diff
@@ -34,6 +34,7 @@ function Shape(shape, options, memberName) {
   property(this, 'isLocationName', Boolean(shape.locationName), false);
   property(this, 'isIdempotent', shape.idempotencyToken === true);
   property(this, 'isJsonValue', shape.jsonvalue === true);
+  property(this, 'isSensitive', shape.sensitive);
```

Comment:
Could you cast to a boolean? (`shape.sensitive === true`)

Reply:
Actually, this may lead to wrong output. For some shape, it' sensitive if its prototype (like 'S19') is sensitive. I think `shape.sensitive === true || shape.prototype && shape.prototype.sensitive === true` can be a workaround.